kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaTopicManager.java

```diff
@@ -371,8 +347,8 @@ public synchronized void close() {
                     topicFuture.get().removeProducer(references.get(topicName));
```

Comment:
`topicFuture.get()` needs a null check because the topic future that was put to `topics` may complete with null.